_data/simple-icons.json

```diff
@@ -11038,6 +11038,11 @@
             "hex": "0A53E0",
             "source": "https://www.opslevel.com"
         },
+        {
+            "title": "Optimism",
+            "hex": "FF0420",
+            "source": "https://optimism.io"
```

Comment:
I'm not seeing an SVG available here? Where are you getting it from?

Reply:
I just used your suggested svg from https://github.com/simple-icons/simple-icons/issues/10779#issuecomment-2094949398 Didn't realize that it wasn't from the official website. So, I should put the GitHub repo as source?

Reply:
See https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines.

Reply:
Ok, I updated the source to https://github.com/ethereum-optimism/brand-kit/blob/main/assets/svg/Profile-Logo.svg.